include/wx/uilocale.h

```diff
@@ -38,19 +45,31 @@ class WXDLLIMPEXP_BASE wxUILocale
     // Get the object corresponding to the currently used locale.
     static const wxUILocale& GetCurrent();
 
+    // Create the object corresponding to the given locale.
+    explicit wxUILocale(const wxLocaleIdent& localeId);
+
+    // Check if the locale is actually supported by the current system: if it's
+    // not supported, the other functions will behave as for the "C" locale.
+    bool IsSupported() const;
+
     // Get the platform-dependent name of the current locale.
     wxString GetName() const;
 
     // Query the locale for the specified information.
     wxString GetInfo(wxLocaleInfo index,
                      wxLocaleCategory cat = wxLOCALE_CAT_DEFAULT) const;
 
+    // Compares two strings, for a locale specified by wxLocaleIdent.
```

Comment:
Does the "for a locale specified by wxLocaleIdent" part really belong there?

Reply:
No, it's a leftover from the original version in which it was static, thanks for noticing!